### PR TITLE
chore: assign googleapis/cdpe-cloudai as codeowner

### PR DESCRIPTION
**Pull request by parthea**

### CODE REVIEW

**Review by busunkim96 — APPROVED** [review]

File: .repo-metadata.json [hunk 1]
```diff
@@ -12,5 +12,5 @@
     "api_id": "documentai.googleapis.com",
     "requires_billing": true,
     "default_version": "v1",
-    "codeowner_team": "@googleapis/ml-apis"
+    "codeowner_team": "@googleapis/ml-apis @googleapis/cdpe-cloudai"
```
Comment (busunkim96) [inline .repo-metadata.json:15]: Is having both teams here intended?

Comment (parthea) [reply .repo-metadata.json:15]: I'll remove googleapis/ml-apis

**Comment by parthea** [comment]
@busunkim96 Please could you take another look?

**Review by busunkim96 — APPROVED** [review]

**Comment by parthea** [comment]
The samples failures are tracked in https://github.com/googleapis/python-documentai/issues/244. I'm going to merge this PR as the changes are unrelated to samples testing.

**Pull request merged**